logs_api/lib/opentelemetry-logs-api.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require_relative 'opentelemetry/logs'
```

Comment:
I think you could add a require for the version file here as well since that file exists. Currently I'm only seeing it required in the gemspec.

Reply:
Fixed in: c0308e480b326945409ea0206250941cf6f7a9dc